js/deck.js

```diff
@@ -0,0 +1,51 @@
+class Deck {
+    constructor(size) {
+        this.size = size;
+        this.cards = []
+    }
+        newDeck() {
+            const names = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K', 'A'];
+            const suits = ['Hearts','Diamonds','Spades','Clubs'];
```

Comment:
Te jādomā, kur labāk šo varētu nodefinēt. It kā varētu ienest konstruktorā. Varētu arī iznest ārpus klases. Ceturtdien varam padiskutēt. Pagaidām lai paliek.